### PR TITLE
style and whitespace changes before fixing endpoint logic

### PR DESCRIPTION
**Pull request by hornc**
Relates to #2857

This is to get all the whitespace and quotation mark changes I'd make as part of this work out and merged in one go.

Double spaces between classes etc help me navigate the file faster, and are in PEP8 https://www.python.org/dev/peps/pep-0008/#blank-lines

### CODE REVIEW

File: openlibrary/plugins/openlibrary/code.py [hunk 1]
```diff
@@ -44,12 +44,12 @@
     api = None
 
 # http header extension for OL API
-infogami.config.http_ext_header_uri = "http://openlibrary.org/dev/docs/api"
+infogami.config.http_ext_header_uri = 'http://openlibrary.org/dev/docs/api'
```
Comment (hornc) [inline openlibrary/plugins/openlibrary/code.py]: This should be https I guess, although I have not confirmed this is being used

**Comment by cdrini** [comment]
❤️ ! Thank you for putting this in a separate PR :)

(In case others are curious, here's the command to make this happen automatically; from https://lintlyci.github.io/Flake8Rules/ )

```
pip install autopep8
autopep8 --in-place --select=E305 openlibrary/plugins/openlibrary/code.py # 2 blank lines after class
```

Quick question (not blocking), but one of the biggest python formatter, black, has landed on double-quotes as their standard. I think it might make more sense to use that instead. Thoughts? cc: @cclauss

**Comment by cclauss** [comment]
The Python Software Foundation is recommending psf/black.

**Comment by hornc** [comment]
@cdrini Insisting on double quotes would make me sad. I honestly thought there was a style guide that favoured single over double quotes, but there is no PEP which says that, and the Google style guide doesn't favour one over the other either. I may have picked it up from another prominent style guide, but if I did it's not prominent enough for me to find it again.

PEP8 says choose one and stick to it, I have clearly chosen single, and I thought it was objectively more 'Pythonic', but I can't find a concrete defence for that provided by anyone else online.

_My_ reason why I thought it was the proper way, is that when you ask Python itself for a representation of a string, it shows single quotes:

```
>>> quotetest = ['a', "b", '''c''', """d""", "This is a longer string with human readable content!\nAnd an newline."] 
>>> quotetest
['a', 'b', 'c', 'd', 'This is a longer string with human readable content!\nAnd an newline.']
>>> print(quotetest)
['a', 'b', 'c', 'd', 'This is a longer string with human readable content!\nAnd an newline.']
``` 

I thought that was pretty compelling, and was Python itself indicating how strings should be represented.

I may have originally picked up this impression from Python object serialisation using Pickle or similar, but I can't remember.

**Comment by cclauss** [comment]
Fascinating that the new version of both the JavaScript and Python linters are pushing double quotes.  My feeling on Google is that they got to specify the format behind gofmt but I see no reason why they should be the authority on Python formatting especially when they prefer two space indentation over four spaces.  I will track with the Python Software Foundation on my projects.

**Comment by cdrini** [comment]
Yeah; my guess would be you may have also been influenced by our JS options, which enforce single quotes. I was looking for an `autopep8` command to auto fix the quotes, but it didn't exist :P I'm going to open a separate issue for reaching a consensus on this and merge this PR though, since I imagine you have more work waiting. Regardless of what we end up choosing, it'll should be automatic, so it'll be easy to apply in bulk later :+1:

**Comment by cdrini** [comment]
Threw on dev and did some sanity checks that everything was working ok. Tested works pages, editions pages, loans page, subjects page, routes pages.

**Comment by tfmorris** [comment]
> I'm going to open a separate issue for reaching a consensus on this and merge this PR though,

But, but, that's the worst of all possible outcomes - gratuitous style (ie quote) changes to satisfy a personal preference. When the next editor changes them all back, we'll have an endless sea of noisy diffs.

If there's not an established style, the style in use already in the module should be the default.

**Comment by cdrini** [comment]
My guess would be Charles had already started making changes on top of this, and a refactor would have been really time consuming/difficult. It's also pretty trivial to skip a commit when git blaming a file. This PR was grandfathered in, essentially; I think avoiding big style changes until there's a consensus makes sense though (although the 2 line stuff seems ok, since there's no contention).

**Pull request merged**